library/common/src/main/java/com/epmedu/animeal/common/constants/DataStorePreferencesKey.kt

```diff
@@ -4,8 +4,8 @@ import androidx.datastore.preferences.core.stringPreferencesKey
 
 object DataStorePreferencesKey {
     val phoneNumberKey = stringPreferencesKey("phone_number")
-    val firstNameKey = stringPreferencesKey("first_name")
-    val lastNameKey = stringPreferencesKey("last_name")
+    val nameKey = stringPreferencesKey("name")
```

Comment:
Just a reminder, it will break the existing profile on the device. During development it's ok, but be careful letter 🙂

Reply:
> Just a reminder, it will break the existing profile on the device. During development it's ok, but be careful letter 🙂

Yeah, I know)